CODING_GUIDELINES.md

```diff
@@ -1,10 +1,10 @@
-﻿All new code **must** follow the following coding guidelines.  
```

Comment:
Trimming those whitespaces changed the result layout. 
Before: https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md
After: https://github.com/qbittorrent/qBittorrent/blob/e1fa7520721880447884e2d3f8591ed732a2bd61/CODING_GUIDELINES.md
You should replace them with a trailing `\` instead.

Reply:
fixed.